chart/postgres-operator/templates/service-account.yaml

```diff
@@ -0,0 +1,12 @@
+{{ if .Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "serviceAccountName" . }}
```

Comment:
Do we need the service account name to be customisable independenty? 
Can't we use the name of the chart instance from `helm install --name my-name-aaa` . If the name is not supplied, helm generates one automatically.

Reply:
That's the idea of the `serviceAccountName` helper does, basically. I'm improving it atm, so that it anloy asks for creation and uses `default` otherwise. So no custom name anymore...

Reply:
@ieugen Could you have a look again at the helper, I just fixedup-rebased...

Reply:
I've tried the cluster again and it did not start, same issue it seems: 
````
github.com/crunchydata/postgres-operator/controller/clustercontroller.go:82: Failed to list *v1.Pgcluster: pgclusters.cr.client-go.k8s.io is forbidden: User "system:serviceaccount:postgres-operator:default" cannot list pgclusters.cr.client-go.k8s.io in the namespace "postgres-operator"
````

Reply:
I was wondering, as I did not found any reference to `serviceaccount` in no .go file...
Either the service account is intelligently detected by some implicit logic in some dependency or it just takes the first available service account in the namespace, which in my (and your case) is default... @jmccormick2001 Are we on the right track?